package.json

```diff
@@ -128,8 +150,10 @@
         "uuid": "^3.3.2",
         "vscode-azureextensionui": "0.26.3",
         "vscode-extension-telemetry": "0.0.18",
+        "vscode-languageclient": "^5.2.1",
         "vscode-nls": "3.2.4",
-        "vscode-uri": "1.0.6"
+        "vscode-uri": "1.0.6",
```

Comment:
where is this used?

Reply:
It is used in `modaRepositoryAnalysisClient.ts`.